cfme/networks/views.py

```diff
@@ -798,3 +800,16 @@ class entities(View):  # noqa
     def is_displayed(self):
         return (self.navigation.currently_selected == ['Networks', 'Floating IPs'] and
                 self.title.text == '{} (Summary)'.format(self.context['object'].address))
+
+
+class OneProviderFloatingIpView(BaseLoggedInPage):
+    """ Represents Floating Ip all for specific Network provider """
+    toolbar = View.nested(OneProviderComponentsToolbar)
+    including_entities = View.include(FloatingIpEntities, use_parent=True)
+
+    @property
+    def is_displayed(self):
+        title = '{name} (All Floating IPs)'.format(name=self.context['object'].name)
+        return (super(BaseLoggedInPage, self).is_displayed and
```

Comment:
Consider calling `self.logged_in_as_current_user` directly instead of the call to `super().is_displayed`